pkgs/development/compilers/symbiflow-yosys-plugins/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, symbiflow-yosys
+, zlib
+, readline
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "symbiflow-yosys-plugins";
+  version = "1.0.0.7-0060-g7454cd6";
+
+  src = fetchFromGitHub {
+    owner  = "SymbiFlow";
+    repo   = "yosys-symbiflow-plugins";
+    rev    = "7454cd6b5e4fd22854e2ada219a5e3c3a06e0717";
+    sha256 = "0r9r31p7fy4ylfrwvwlbivq5a03xrph34blxbxzx2c8bc02mbv0s";
+  };
+
+  patches = [
+    ./make.patch
+  ];
+
+  stripLen = 1;
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ symbiflow-yosys ];
```

Comment:
Likely belongs to `buildInputs` instead.

Reply:
As I recall it's used to build some verilog files. It's not used in actual linking.